twingate/post_install.md

```diff
@@ -0,0 +1,33 @@
+## Twingate Connector Settings
+
+In order to configure and use the Twingate Connector, you first need to create a Twingate Network. You can [sign up](https://www.twingate.com) for a free trial account on our website. The [first-time configuration guide](https://docs.twingate.com/docs/twingate-configuration) covers the necessary steps for the initial configuration of your network. For more information on Twingate, please see our [documentation](https://docs.twingate.com/docs/how-twingate-works).
+
+Twingate Connector settings are passed via a ConfigMap and a Secret, as detailed below.
+
+### ConfigMap settings
+
+This will help you to create a valid configuration. You only need to replace `example.twingate.com` in the snippet below with your Twingate Network fully-qualified domain name, which is configured when you create your Twingate Network.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: connector
+data:
+  url: https://example.twingate.com
```

Comment:
When you create a Civo Kubernetes cluster you get a DNS name that can be used in the marketplace via CIVO:CLUSTER_ID
Can we use that and create that for user ? so that its more of 1 click experience ?

Reply:
The example part of the url is the tenant slug in Twingate. It is not related to created cluster, you can have multiple clusters in civo and other cloud providers connect to the same network.

Reply:
okay